source/includes/harvest/_scheduled_interviews.md

```diff
@@ -151,8 +160,11 @@ Parameter | Description
 | starts_after | Only return scheduled interviews scheduled to start at or after this timestamp.
 | ends_before | Only return scheduled interviews scheduled to end before this timestamp.
 | ends_after | Only return scheduled interviews scheduled to end at or after this timestamp.
+| external_event_id | Only return the scheduled interview that has the specified `external_event_id`.
+| actionable | Only certain scheduled interviews can be updated/deleted. They must have been created through harvest, have a status of "scheduled" or "awaiting feedback," and belong to an active application. When set to true, this filter will return only scheduled interviews that meet those criteria.
```

Comment:
Looks like we capitalize `harvest` everywhere else in the docs right now.

Reply:
kk